libs/cases/frontend/view/src/lib/components/cases-brows/case-brows.component.ts

```diff
@@ -0,0 +1,50 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/core';
+import { RouterLink } from '@angular/router';
+import { IxModule } from '@siemens/ix-angular';
+
+import { ICaseMock } from '../case.mocks/case.interface';
+import { cases } from '../case.mocks/const';
+
+@Component({
+	selector: 'lib-open-cases',
+	standalone: true,
+	imports: [ CommonModule, IxModule, RouterLink ],
+	templateUrl: './case-brows.component.html',
+	styleUrl: './case-brows.component.scss',
+	encapsulation: ViewEncapsulation.None,
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class CaseBrowsComponent {
+    protected readonly _cases = cases;
+    sortedCases: ICaseMock[];
+
+    ngOnInit(): void {
+        const statusOrder = [ 'OPEN', 'INPROGRESS', 'OVERDUE', 'ONHOLD', 'DONE', 'CANCELLED', 'ARCHIVED' ];
```

Comment:
Why dont you add an enum to increase typesafetiness?

https://github.com/UselessPickles/ts-enum-util/blob/HEAD/docs/EnumWrapper.md#get-lists-of-enum-data

Example:
```ts
// import the $enum helper function
import { $enum } from "ts-enum-util";

// Example string enum
// (basic numeric enums also supported)
// (enums with a mix of numeric and string values also supported)
enum RGB {
    R = "r",
    G = "g",
    B = "b"
}

// type: ("R" | "G" | "B")[]
// value: ["R", "G", "B"]
const keys = $enum(RGB).getKeys();

// type: RGB[]
// value: ["r", "g", "b"]
const values = $enum(RGB).getValues();

// List of key/value pair tuples
// type: [("R" | "G" | "B"), RGB][]
// value: [["R", "r"], ["G", "g"], ["B", "b"]]
const entries = $enum(RGB).getEntries();
```

Use `npm i -s ts-enum-util` to install

Reply:
Enums are planned (branch 137). Originally they were another ticket and I looked into it but other devs have been working with the current strings.
I fear that this will cause a lot of merge conflicts and I would rather take the time next sprint to change to enums then to change it here and try to resolve a load of conflicts until tomorrow.

Reply:
good to know